com.unity.shadergraph/Documentation~/index.md

```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
 
 For an introduction to Shader Graph, see [Getting Started](Getting-Started.md).
 
-Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. If you install a prebuilt Scriptable Render Pipeline (SRP) such as the [Universal Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest) (URP) or the [High Definition Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest) (HDRP), Unity automatically installs Shader Graph in your project.
+Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. 
 
 Shader Graph package versions on Unity Engine 2018.x are *Preview* versions, which do not receive bug fixes and feature maintenance. To work with an actively supported version of Shader Graph, use Unity Engine 2019.1 or higher.
+
+## SRP packages are part of the core 
+
+With the release of Unity 2021.1, graphics packages are relocating to the core of Unity. This move simplifies the experience of working with new Unity graphics features, as well as ensuring that your projects are always running on the latest verified graphics code.
+
+For each release of Unity (alpha / beta / patch release) the graphics code is embedded within the main Unity installer. When you install the latest release of Unity, you also get the latest [Universal Render Pipeline (URP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest), [High Definition Render Pipeline (HDRP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest), Shader Graph, [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest), and more.
```

Comment:
"the graphics code is embedded within the main Unity installer. "
I propose
"graphics packages are included in the Unity installer."

and 
" [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest), and more."
-->
" and [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest) packages, among others."

Reply:
I didn't write this part - this was what was provided by the TPMs as something that apparently they want to be consistent across packages. It was just a cut and paste, so I don't know if we want to mess with it (though I admit I don't like how they wrote it, either)